src/reference/00-Getting-Started/02-sbt-by-example.md

```diff
@@ -211,35 +226,34 @@ sbt:Hello> ~testQuick
 
 ### Write a test
 
-Leaving the previous command running, create a file named `src/test/scala/HelloSpec.scala`
+Leaving the previous command running, create a file named `src/test/scala/example/HelloSpec.scala`
```

Comment:
Tiny change, but I noticed it while going through this. Since we had the user create `src/main/scala/example/` I thought it appropriate to also do the same for the test, which is more than likely going to be followed on their teams anyways.